frontend/app/src/screens/schema/schema-viewer.tsx

```diff
@@ -200,6 +200,7 @@ const Properties = ({ schema }: { schema: IModelSchema }) => {
         <PropertyRow title="Default filter" value={schema.default_filter} />
         <PropertyRow title="Order by" value={schema.order_by} />
         <PropertyRow title="Uniqueness constraints" value={schema.uniqueness_constraints} />
+        <PropertyRow title="Human Friendly ID" value={schema.human_friendly_id} />
```

Comment:
code is correct. I was just wonder why placing it here instead of the first section next to ID for example?

Reply:
Ok, that makes sense. I added it a bit below the ID as the types of IDs are a bit different but it's in the top section now.